docs/quickstart.qbk

```diff
@@ -0,0 +1,288 @@
+[/=============================================================================
+    Copyright (C) 2018 Mikael Simberg
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+=============================================================================/]
+
+[section:applications Quick Start]
+
+This section is intended to get you to the point of running a basic __hpx__
+program as quickly as possible. To that end we skip many details but instead
+give you hints and links to more details along the way.
+
+We assume that you are on a Unix system with access to reasonably recent
+packages. You should have `cmake` and `make` available for the build system
+(`pkg-config` is also supported, see __pkg_config__).
+
+[heading Getting HPX]
+
+Download a tarball of the latest release from __stellar_hpx_download__ and
+unpack it or clone the repository directly using `git`:
+
+``
+    git clone https://github.com/STEllAR-GROUP/hpx.git
+``
+
+It is also recommended that you check out the latest stable tag:
+
+``
+    git checkout 1.1.0
```

Comment:
We should add a note to the release-procedure document to not to forget changing this version (here: 1.1.0) during release preparations.

Reply:
I'm counting on point 26 (the note about grepping old version numbers) in the [release procedure](https://github.com/STEllAR-GROUP/hpx/blob/master/docs/release_procedure.rst) catching that.

We could also have a `latest` tag or something like that which would always point to the latest release and say `git checkout latest`, but I'm not sure if that's a good idea (tags shouldn't change and so on...).

Reply:
Ok, thanks!